### PR TITLE
Monday-09-25

### PR DESCRIPTION
**Pull request by miteshqodic**
Monday-09-25

### CODE REVIEW

File: src/Message/SendEmailConfirmationAndResetPassword.php
Comment (illuminate3) [inline src/Message/SendEmailConfirmationAndResetPassword.php:1]: Why was this file deleted and not the one with the word Buyer in it?

File: src/MessageHandler/SendEmailConfirmationAndResetPasswordLinkHandler.php
Comment (illuminate3) [inline src/MessageHandler/SendEmailConfirmationAndResetPasswordLinkHandler.php:1]: Why was this file deleted and not the one with the word Buyer in it?

File: src/Service/Auth/EmailVerifierAndResetPasswordService.php
Comment (illuminate3) [inline src/Service/Auth/EmailVerifierAndResetPasswordService.php:1]: Why was this file deleted and not the one with the word Buyer in it?

**Review by illuminate3 — CHANGES_REQUESTED** [review]
I don't see why we keep the files with the word Buyer in them.
I do know why it would be an easy choice to do that.

Should we add Buyer to all files?

The decision should be made as to which one would provide cleaner and easier to understand code.

I spend hours cleaning up and reorganizing code. I don't want to keep doing that. Let's please try to clean up the code.

**Comment by miteshqodic** [comment]
> I don't see why we keep the files with the word Buyer in them. I do know why it would be an easy choice to do that.
> 
> Should we add Buyer to all files?
> 
> The decision should be made as to which one would provide cleaner and easier to understand code.
> 
> I spend hours cleaning up and reorganizing code. I don't want to keep doing that. Let's please try to clean up the code.

Initially, we have the filename without the buyer word, then we combined dealer and buyer code.
then we decided to change the flow and buyer and dealer should be separate project, so files exists since that time.

Ok no worries, I will remove buyer word files and user without that and make appropriate changes and push

**Review by illuminate3 — APPROVED** [review]

**Pull request merged**